packages/namekit-react/src/components/Link.tsx

```diff
@@ -3,7 +3,7 @@ import cc from "classcat";
 
 export interface LinkProps extends React.ComponentPropsWithoutRef<"a"> {
   className?: string;
-  variant?: "primary" | "secondary" | "underline";
+  variant?: "primary" | "secondary" | "ghost" | "underline";
```

Comment:
@notrab Whenever we make an update like this please also proactively update the storybook for the related UI component.

The storybook for both `Link` and `Button` are a mess right now too.

![CleanShot 2024-09-11 at 18 07 51](https://github.com/user-attachments/assets/236d4a87-a667-4adc-b519-64b975cfb297)

For example, I can't customize the variant or the size as expected.

And what are the `rel` and `target` controls for? If these are exposed via Storybook then I should be able to figure out what these do just from the example stories without needing to read any of the code. But there aren't any stories that communicate any ideas about these props. So we should either create those stories or remove these props as controls in Storybook.

![CleanShot 2024-09-11 at 18 12 15](https://github.com/user-attachments/assets/1c973998-0426-4d07-bd2a-80e93e14aefc)

The above is also broken. Can you please investigate why? I see that `colorful-text` works on the story for `TruncatedText`.

Reply:
@lightwalker-eth this is the first time I have seen these "controls". I had them disabled when I first implemented Storybook, so I had no idea they were even here.